com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -279,7 +279,7 @@ public byte[] constructLuaJITBytecode(Task<Void> task, String luajitExe, String
         options.add(Bob.getExe(Platform.getHostPlatform(), luajitExe));
         options.add("-b");
         options.add("-g"); // Keep debug info
-        options.add("-f"); options.add(chunkName);
+        options.add("-F"); options.add(task.input(0).getPath()); // The @ is added in the tool
```

Comment:
This is one change we've managed to "merge" upstream:
https://github.com/LuaJIT/LuaJIT/blob/v2.1/src/jit/bcsave.lua#L679-L680

Reply:
Nice!